core/src/components/radio-group/radio-group.tsx

```diff
@@ -64,6 +94,35 @@ export class RadioGroup implements ComponentInterface {
     }
   }
 
+  @Listen('keydown', { target: 'document' })
+  onKeydown(ev: any) {
```

Comment:
Can we do `if (ev.target && !this.el.includes(ev.target)) { return; }`? That way if we have 10 radio groups, we aren't going to do 10 `querySelectorAll` calls. It would be good to make this a single listener, but not sure that's possible since this callback seems dependent on the radio group it is in.